data/core/encyclopedia/geography.cfg

```diff
@@ -30,33 +30,58 @@
 [topic]
     id=great_river
     title= _ "Great River"
-    text= _ "The Great River forms the boundary between the <ref>dst='kingdom_wesnoth' text='Kingdom of Wesnoth'</ref> and the <ref>dst='northlands' text='Northlands'</ref>. It empties to the <ref>dst='great_ocean' text='ocean'</ref> at <ref>dst='elensefar' text='Elensefar'</ref>."
+    text= _ "The Great River forms the boundary between the <ref>dst='kingdom_wesnoth' text='Kingdom of Wesnoth'</ref> and the <ref>dst='northlands' text='Northlands'</ref>. It empties to the <ref>dst='great_ocean' text='ocean'</ref> at <ref>dst='elensefar' text='Elensefar'</ref>." + "
+
+" + _ "The Ford of Abez is a crossing between Wesnoth and the Northlands, and is the westernmost point where the Great River can be crossed without ships."
 [/topic]
 
 [topic]
     id=great_ocean
     title= _ "Great Ocean"
-    text= _ "Lies to the west of the <ref>dst='great_continent' text='continent'</ref> and all rivers eventually flow to it. Far to the west in the Great Ocean is a huge archipelago called <ref>dst='morogor' text='Morogor'</ref>."
+    text= _ "Lies to the west of the <ref>dst='great_continent' text='continent'</ref> and all rivers eventually flow to it. Far to the west in the Great Ocean is a huge archipelago called <ref>dst='morogor' text='Morogor'</ref>." + "
+    
+" +
+        #po: Great Ocean
+        _ "The Drakes once called this ocean the World Ocean. They believe it to extend in every direction from Morogor until eventually flowing into the abyss as an immense waterfall at the edge of the world."
 [/topic]
 
 [topic]
     id=morogor
     title= _ "Morogor"
     text= _ "Archipelago, located somewhere in the <ref>dst='great_ocean' text='Great Ocean'</ref> east of the <ref>dst='green_isle' text='Green Isle'</ref> and west of the <ref>dst=great_continent text='Great Continent'</ref>.
 It is mostly inhabited by <ref>dst='..race_drake' text='drakes'</ref>.
-The central island of the archipelago is also called ‘Morogor’."
+The central island of the archipelago is also called ‘Morogor’." + "
+
+" +
+        #po: Geography of Morogor
+        _ "<header>text='Notable land features:'</header>
+               • Mount Morogor: An active volcanic mountain in the central island of Morogor."
 [/topic]
 
 [topic]
     id=green_isle
     title= _ "Green Isle"
-    text= _ "A bigger island lying in the <ref>dst='great_ocean' text='Great Ocean'</ref>."
+    text= _ "A bigger island lying in the <ref>dst='great_ocean' text='Great Ocean'</ref>, formerly home to the Islefolk and Wesfolk." + "
+
+" +
+        #po: Geography of the Green Isle
+        _ "<header>text='Notable cities:'</header>
+               • Southbay: The capital of the largest Islefolk Kingdom.
+               • Clearwater Port: The second largest Islefolk city and the location of a major port.
+               • Stormvale: A major Islefolk Kingdom once ruled by the ancestors of Haldric I.
```

Comment:
Shouldn't 'kingdom' be lower-case in this instance?

Reply:
A follow-up on some small revision will probably be necessary. I planned on adding Southport and removed Southwood from this due the TSG Re-Revision as well, so I might open a new PR with that, together with fixing these small mistakes.

Reply:
Just so you're aware (I wasn't expecting anyone to make such a quick response), I'll make a PR to fix what I'm finding when I've finished the revision (I'm finding these as a translator/localiser).

Reply:
Thanks! I will still be making a follow-up, but probably only once the TSG Re-Revision is merged.